migrations/0001.mine-record.sql

```diff
@@ -1,27 +1,33 @@
-CREATE TABLE mine_identifier (
+CREATE TABLE mine_identity (
   mine_guid        uuid PRIMARY KEY DEFAULT gen_random_uuid(),
   create_user      character varying(60) NOT NULL DEFAULT 'DUMMY_USER',
   create_timestamp timestamp with time zone NOT NULL DEFAULT current_timestamp,
   update_user      character varying(60) NOT NULL DEFAULT 'DUMMY_USER',
   update_timestamp timestamp with time zone NOT NULL DEFAULT current_timestamp
 );
 
-CREATE TABLE mine_details (
-  mine_guid uuid,
+CREATE TABLE mine_detail (
+  mine_detail_guid uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  mine_guid uuid NOT NULL,
   mine_no   character varying(010) NOT NULL,
   mine_name character varying(100) NOT NULL,
+
+  effective_date date NOT NULL DEFAULT now(),
+  expiry_date    date     NULL DEFAULT NULL,
   create_user      character varying(60) NOT NULL DEFAULT 'DUMMY_USER',
```

Comment:
I know its not part of this PR, but i don't think we want a default 'DUMMY_USER' here? we should just leave it as NOT NULL, i will handle defaults for tests and such at the API level.

Reply:
@garywong-bc